content/pt-br/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Limites e reservas de ID de processo
```

Comment:
Perhaps adding "(PIDs)" at the end of the title here like you did on line 11 would be a nice addition.

Reply:
Done